IntersonArrayDeviceRF.hxx

```diff
@@ -479,7 +479,7 @@ private:
     {
     for( unsigned int i = 0; i < bModeRingBuffer.size(); i++ )
       {
-      ImageType::Pointer image = ImageType::New();
+      bModeRingBuffer[ i ] = ImageType::New();
```

Comment:
I don't think there is a bug. A new image is/was allocated for each i within the loop in both versions.